app/models/form/lettings/questions/tenancy_type.rb

```diff
@@ -0,0 +1,37 @@
+class Form::Lettings::Questions::TenancyType < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "tenancy"
+    @check_answer_label = "Type of main tenancy"
+    @header = "What is the type of tenancy?"
+    @type = "radio"
+    @check_answers_card_number = 0
+    @hint_text = ""
+    @answer_options = ANSWER_OPTIONS
+    @conditional_for = { "tenancyother" => [3] }
+  end
+
+  ANSWER_OPTIONS = {
+    "4" => {
+      "value" => "Assured Shorthold Tenancy (AST) – Fixed term",
+      "hint" => "Mostly housing associations provide these. Fixed term tenancies are intended to be for a set amount of time up to 20 years.",
```

Comment:
is there a consistent approach when writing fixed term vs fixed-term?

Reply:
Not that I'm aware of. Just copying copy given in ticket